src/client/pydaos/SConscript

```diff
@@ -31,7 +34,7 @@ def build_shim_module(py_env, svars, lib_prefix, version):
                                CPPFLAGS=svars["BASECFLAGS"] + " " + \
                                svars["OPT"] + " " + extra_flags)
     base = daos_build.library(new_env, target=base_name, source=[obj],
-                              LIBS=['daos'],
+                              LIBS=libraries,
```

Comment:
since daos_build.library only adds ORIGIN to RPATH and since it's different for python module, I would just use new_env.SharedLibrary instead here